conda/cpuonly/meta.yaml

```diff
@@ -2,7 +2,13 @@ package:
   name: cpuonly
   version: 1.0
 
-build:
-  track_features:
-      - cpuonly
-  noarch: generic
+requirements:
+  run_constrained:
+    - pytorch-proc * cpu
+
+outputs:
+  # A meta-package to select CPU or GPU build for faiss.
```

Comment:
This comment is a little confusing - why specifically for `faiss`? I assume it's for `pytorch` and possibly other packages too?

Reply:
Oopps, this change shouldn't have made it in